integration/integration_suite_test.go

```diff
@@ -89,3 +89,8 @@ func startGinkgo(dir string, args ...string) *gexec.Session {
 	Ω(err).ShouldNot(HaveOccurred())
 	return session
 }
+
+func removeSuccessfully(path string) {
```

Comment:
Can we just use `Expect(os.RemoveAll(path)).To(Succeed())` ? Do we need this extra indirection to another method as well, or is it cause you don't want the assertion in the test block itself?

Reply:
Yes, the initial idea was not to have the assertion within the block itself. The PR is still WIP since we started refactoring to make sure all generated coverage files are always removed.